cmd/kube-proxy/app/server_test.go

```diff
@@ -219,6 +219,30 @@ nodePortAddresses:
 			healthzBindAddress: "[fd00:1::5]:12345",
 			metricsBindAddress: "[fd00:2::5]:23456",
 		},
+		{
+			// Test for unknown field within config.
+			// For v1alpha1 a lenient path is implemented and will throw a
```

Comment:
If you need to specify the version, we can update the `yamlTemplate`.

Reply:
Based on our chat earlier today, I think we have a gameplay for further cleanup around this and that will come as a separate PR.